### PR TITLE
Allows non admins to use the block

### PR DESCRIPTION
**Pull request by roborourke**
Bit hacky but this is an improvement in terms of allowing non admins to use the block.

### CODE REVIEW

File: embed-gutenberg-block-google-maps.php [hunk 1]
```diff
@@ -70,6 +75,32 @@ function register_settings() {
 
 add_action( 'init', __NAMESPACE__ . '\\register_settings' );
 
+function admin_settings() {
+	add_settings_section( __NAMESPACE__, __( 'Maps Block Settings' ), '__return_false', 'writing' );
+	add_settings_field( 'pantheon_google_map_block_api_key', __( 'API Key' ), function () {
+		printf( '<input type="text" name="%s" class="widefat regular-text" value="%s" %s />',
+			'pantheon_google_map_block_api_key',
+			get_google_api_key(),
```
Comment (joemcgill) [inline embed-gutenberg-block-google-maps.php]: esc_attr

File: src/js/EditorBlock.js [hunk 2]
```diff
@@ -10,67 +11,31 @@ import classnames from 'classnames';
...
-				isLoading: false,
-				isSaving: false,
-				keySaved: true,
-			} );
-			settings.fetch();
-		} );
 	}
 
 	render() {
+		console.log( this.state, this.state.apiKey === '' )
```
Comment (joemcgill) [inline src/js/EditorBlock.js]: Leftover from debugging?

**Review by joemcgill — CHANGES_REQUESTED** [review]
A couple notes, otherwise looks good.

File: src/js/EditorBlock.js (hunk 2 above)
Comment (roborourke) [reply src/js/EditorBlock.js]: whoops!

File: embed-gutenberg-block-google-maps.php (hunk 1 above)
Comment (roborourke) [reply embed-gutenberg-block-google-maps.php]: classic

Comment (roborourke) [reply embed-gutenberg-block-google-maps.php]:
```suggestion
			esc_attr( get_google_api_key() ),
```

File: src/js/EditorBlock.js (hunk 2 above)
Comment (roborourke) [reply src/js/EditorBlock.js]:
```suggestion
```

**Pull request merged**